onnx/cpp2py_export.cc

```diff
@@ -623,12 +623,24 @@ PYBIND11_MODULE(onnx_cpp2py_export, onnx_cpp2py_export) {
         ModelProto model{};
         ParseProtoFromPyBytes(&model, bytes);
         auto function_id_set = inliner::FunctionIdSet::Create(std::move(function_ids), exclude);
-        inliner::InlineSelectedFunctions(model, *function_id_set);
+        inliner::InlineSelectedLocalFunctions(model, *function_id_set);
         std::string out;
         model.SerializeToString(&out);
         return py::bytes(out);
       });
 
+  inliner.def(
+    "inline_selected_functions2",
```

Comment:
What's the motivation behind a new API? Can we instead extend `inline_selected_functions` to also handle schema functions as well?

Reply:
Also please see Justin's comment above ... relates to python backward-compatibility

Reply:
Ah thanks for clarifying. Makes sense.